Django Selenium Initial Test Scripts

akbranam opened the pull request.

akbranam committed:
Update README.md

akbranam committed:
Update README.md

akbranam committed:
Update README.md

akbranam committed:
Update README.md

akbranam committed:
Update README.md

akbranam committed:
Initial test scripts

Initial scripts for testing django portal with selenium. Tests include: experiment creation and subbmission, experiment cloning, experiment cancelation, user account creation, user logout, and load testing.

akbranam committed:
Merge branch 'master' of https://github.com/akbranam/Django-Selenium-Tests

DImuthuUpe commented on org.apache.airavata.django.tests/src/AbinitComet.java:
```diff
@@ -0,0 +1,57 @@
+import static org.junit.jupiter.api.Assertions.*;
```
Add Apache license header as in https://github.com/apache/airavata/blob/master/modules/airavata-helix/helix-spectator/src/main/java/org/apache/airavata/helix/impl/workflow/PostWorkflowManager.java#L1

DImuthuUpe commented on org.apache.airavata.django.tests/.settings/org.eclipse.jdt.core.prefs:
```diff
@@ -0,0 +1,11 @@
+eclipse.preferences.version=1
```
Remove this file. This is specific to local setup. All auto generated files should be removed

DImuthuUpe commented on org.apache.airavata.django.tests/.classpath:
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Remove this

DImuthuUpe commented:
Remove all .class files as they are binaries

akbranam committed:
Updated Test Scripts

Removed generated files
Added Apache Header
Added application creation testing
Updated properties file
